pkg/apis/core/validation/validation.go

```diff
@@ -6839,7 +6839,7 @@ func validateScopedResourceSelectorRequirement(resourceQuotaSpec *core.ResourceQ
 		case core.ResourceQuotaScopeBestEffort, core.ResourceQuotaScopeNotBestEffort, core.ResourceQuotaScopeTerminating, core.ResourceQuotaScopeNotTerminating, core.ResourceQuotaScopeCrossNamespacePodAffinity:
 			if req.Operator != core.ScopeSelectorOpExists {
 				allErrs = append(allErrs, field.Invalid(fldPath.Child("operator"), req.Operator,
-					"must be 'Exist' when scope is any of ResourceQuotaScopeTerminating, ResourceQuotaScopeNotTerminating, ResourceQuotaScopeBestEffort, ResourceQuotaScopeNotBestEffort or ResourceQuotaScopeCrossNamespacePodAffinity"))
+					"must be 'Exists' when scope is any of ResourceQuotaScopeTerminating, ResourceQuotaScopeNotTerminating, ResourceQuotaScopeBestEffort, ResourceQuotaScopeNotBestEffort or ResourceQuotaScopeCrossNamespacePodAffinity"))
```

Comment:
I'm a little confused. Is there anything wrong with Exist?

Reply:
Yes, because user have to use 'Exists' as the value
https://github.com/kubernetes/kubernetes/blob/10ae1dbb52d5d03bcbff981065597e341a3d0237/pkg/apis/core/types.go#L5844
![image](https://github.com/kubernetes/kubernetes/assets/600238/848ab536-3c54-475c-975e-dfe60b86a678)
https://kubernetes.io/docs/concepts/policy/resource-quotas/